autoload/phpactor/input.vim

```diff
@@ -32,19 +33,12 @@ function! phpactor#input#choice(label, choices, ResultHandler)
 
     for choiceLabel in keys(a:choices)
         let buffer = []
-        let foundShortcut = v:false
 
-        for char in split(choiceLabel, '\zs')
-            if v:false == foundShortcut && -1 == index(usedShortcuts, tolower(char))
-                call add(buffer, '&')
-                let foundShortcut = v:true
-                call add(usedShortcuts, tolower(char))
-            endif
-
-            call add(buffer, char)
-        endfor
-
-        let confirmLabel = join(buffer, "")
+        if !empty(a:keyMap) && has_key(a:keyMap, choiceLabel) && !empty(a:keyMap[choiceLabel])
```

Comment:
```suggestion
        if has_key(a:keyMap, choiceLabel) && !empty(a:keyMap[choiceLabel])
```
`v:false == has_key({}, 'x')` is true. No need to check if a:keyMap is empty.

Reply:
`keyMap` can be a `list` when empty due to using an `array` as the JSON source :thinking:

Reply:
Isn't `JSON_FORCE_OBJECT` helpful in such case?

Reply:
Well, we just switch to another problem then. There is probably a way to fix it on the PHP side, but I'm not that concerned.

Reply:
Nothing too relevant to be concerned ;)